lib/edge.js

```diff
@@ -143,7 +156,28 @@ class Edge {
     return this[_name]
   }
 
+  get rawSpec () {
+    return this[_spec]
+  }
+
   get spec () {
+    if (this.overrides && this.overrides.value && this.overrides.name === this.name) {
+      if (this.overrides.value.startsWith('$')) {
```

Comment:
this seems like a strange sentinel/prefix approach. are you envisioning that an override can't ever start with `$`, even tho that might be a legitimate bundledDep folder name inside `node_modules`?

Reply:
[npm-package-arg](https://github.com/npm/npm-package-arg) defines `$` as an invalid character in a package name, anywhere we attempt to parse a spec will explode. i'd be surprised if it worked as a bundled dep name, have you seen it work that way?

i chose the `$` prefix specifically because npm-package-arg will throw when encountering it, but if that assumption doesn't hold water then we'll definitely have to come up with something else

Reply:
nope! just thinking about possibilities.

Will every prior version of npa throw on it?

It's a fine assumption, I'm just not sure why a stringly typed interface is the ideal choice.

Reply:
afaict all version of npa will throw on it.

using a string was a gut call. there are most definitely other approaches we could use here, but this was simple to parse and i knew it would be invalid as a "normal" package spec which is what i wanted.